linux-4.4.defconfig

```diff
@@ -78,7 +78,6 @@ CONFIG_SYN_COOKIES=y
 # CONFIG_IPV6 is not set
 CONFIG_CFG80211=y
 CONFIG_CFG80211_INTERNAL_REGDB=y
-CONFIG_CFG80211_WEXT=y
```

Comment:
I know that it affects a wifi driver on a different platform. I'm not sure if there's a common wifi driver used on the RPi that requires wext. It seems like including wext shouldn't be an issue for 81xx support.